Resources/views/backend/swag_backend_order/controller/main.js

```diff
@@ -617,6 +620,12 @@ Ext.define('Shopware.apps.SwagBackendOrder.controller.Main', {
 
                 me.validationMail = pluginConfigObj.data.validationMail;
                 me.desktopTypes = pluginConfigObj.data.desktopTypes;
+                me.sendMail = pluginConfigObj.data.sendMail;
+                if (me.sendMail == 1) {
+                    me.orderModel.set('sendMail', 1);
+                    //Set checked class for table
+                    document.getElementById('sendMailChk').classList.add('x-form-cb-checked');
```

Comment:
That looks to me totaly wrong way to check a checkbox in extjs :).
Why are you not using setValue method from checkbox? (http://docs.sencha.com/extjs/4.1.0/#!/api/Ext.form.field.Checkbox-method-setValue)